composer.json

```diff
@@ -9,7 +9,7 @@
   "license": "MIT",
   "require": {
     "php": "^7.4 || ^8.0",
-    "sylius/sylius": "^1.10.0"
+    "sylius/sylius": "^1.8.0 || ^1.9.0 || ^1.10.0"
```

Comment:
Same as above, correct constraints should look like that: `"sylius/sylius": "~1.8.0 || ~1.9.0 || ~1.10.0"`